integration/hurl/tests_ok/netrc.py

```diff
@@ -0,0 +1,15 @@
+from flask_httpauth import HTTPBasicAuth
+from app import app
+
+auth = HTTPBasicAuth()
+
+
+@auth.verify_password
+def verify_password(username, password):
+    return username == "Bob" and password == "secret"
+
+
+@app.route("/netrc")
+@auth.login_required
+def netrc():
+    return ""
```

Comment:
Without the "Flask HTTP auth" dependency, the check could simply be:

```python
@app.route("/netrc")
def netrc():
    assert(request.headers["Authorization"] == "Basic Qm9iOnNlY3JldA==")
    return ""
```
As it's done in `tests_ok/basic_authentication.py`

Reply:
Thanks, this is a good idea.